src/components/TransactionFlow/ConfirmTransaction.tsx

```diff
@@ -1,11 +1,9 @@
 import React, { useContext, useEffect, useState } from 'react';
 
-import Styled from 'styled-components';
+import styled from 'styled-components';
 
-import feeIcon from '@assets/images/icn-fee.svg';
-import sendIcon from '@assets/images/icn-send.svg';
-import walletIcon from '@assets/images/icn-wallet.svg';
-import { Amount, AssetIcon, Button, InlineMessage, PoweredByText } from '@components';
+import { Amount, Button, InlineMessage, PoweredByText } from '@components';
+import Icon from '@components/Icon';
```

Comment:
Any reason to not directly import?

Reply:
The imports break when I do.
Most likely due to circular dependencies. eg. A component importing another from the same `index.ts`